src/VisualStudio/Core/Test/Snippets/SnippetTestState.vb

```diff
@@ -55,7 +58,8 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Snippets
                 Dim completionService = DirectCast(Workspace.Services.GetLanguageServices(languageName).GetService(Of CompletionService), CommonCompletionService)
                 completionService.SetTestProviders({snippetProvider})
 
-                _completionCommandHandler = Workspace.GetService(Of CompletionCommandHandler)
+                _completionCommandHandler = GetExportedValues(Of VSCommanding.ICommandHandler)().
+                    Single(Function(e As VSCommanding.ICommandHandler) e.GetType().FullName = "Microsoft.VisualStudio.Language.Intellisense.AsyncCompletion.Implementation.CompletionCommandHandler")
```

Comment:
Should this be finding the command handler with the PredefinedCommandHandlerNames or whatever it is that the editor exposed? That way this doesn't rely on internal implementation details.